llvm/test/ObjectYAML/GOFF/GOFF-Header-End.yaml

```diff
@@ -0,0 +1,18 @@
+# RUN: yaml2obj %s | od -v -An -tx1 | FileCheck --ignore-case %s
+
+# Verify that GOFF Header is correct.
+# CHECK: 03 f0 00 00 00 00 00 00 00 00 00 00 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
+# CHECK: 00 00 01 00 00 00 00 00 00 00 00 00 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
+
+# Verify GOFF Module end.
+# CHECK: 03 40 00 00 00 00 00 00 00 00 00 02 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
+# CHECK: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
```

Comment:
There's nothing in this test that shows you've checked the whole file. I assume that is desirable. One way of doing this is adding `--implicit-check-not={{.}}` to the FileCheck command.

Reply:
This conversation was marked as resolved, but I don't see anything that's changed that would achieve this?

Reply:
I resolved this accidentally. 
When I try to add `--implicit-check-not={{.}}`, my test fails at the first line. The output of `od` shows exactly the same GOFF output. Not sure if I'm missing something obvious here.

Reply:
Hmm, that implies there is some text still printed to the output, but it could be as simple as whitespace, e.g. a blank line at the end or start of the output. Could you pipe the `od` to a file and attach it here, please, and I can then take a quick look?

Reply:
Sure, I've attached it here. 
[output.txt](https://github.com/llvm/llvm-project/files/13652518/output.txt)

Reply:
Could you try adding a `# CHECK-EMPTY:` to the end of the output and see if that fixes it? That's about the only idea I've got, looking at that text file.

Reply:
That seems to work fine.